fdm-core/src/db/migrations/0001_tidy_stranger.sql

```diff
@@ -0,0 +1,166 @@
+CREATE SCHEMA "fdm-dev";
+--> statement-breakpoint
+CREATE TYPE "fdm-dev"."p_app_method" AS ENUM('slotted coulter', 'incorporation', 'injection', 'spraying', 'broadcasting', 'spoke wheel', 'pocket placement');--> statement-breakpoint
+CREATE TYPE "fdm-dev"."b_manage_type" AS ENUM('owner', 'lease');--> statement-breakpoint
+CREATE TYPE "fdm-dev"."b_sector" AS ENUM('diary', 'arable', 'tree_nursery', 'bulbs');--> statement-breakpoint
+CREATE TABLE "fdm-dev"."cultivations" (
+	"b_lu" text PRIMARY KEY NOT NULL,
+	"b_lu_catalogue" text NOT NULL,
+	"created" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated" timestamp with time zone
+);
+--> statement-breakpoint
+CREATE TABLE "fdm-dev"."cultivations_catalogue" (
+	"b_lu_catalogue" text PRIMARY KEY NOT NULL,
+	"b_lu_source" text NOT NULL,
+	"b_lu_name" text NOT NULL,
+	"b_lu_name_en" text,
+	"b_lu_hcat3" text,
+	"b_lu_hcat3_name" text
+);
+--> statement-breakpoint
+CREATE TABLE "fdm-dev"."farm_managing" (
+	"b_id" text NOT NULL,
+	"b_id_farm" text NOT NULL,
+	"b_manage_start" date,
+	"b_manage_end" date,
+	"b_manage_type" "fdm-dev"."b_manage_type",
+	"created" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated" timestamp with time zone,
+	CONSTRAINT "farm_managing_b_id_b_id_farm_pk" PRIMARY KEY("b_id","b_id_farm")
+);
+--> statement-breakpoint
+CREATE TABLE "fdm-dev"."farms" (
+	"b_id_farm" text PRIMARY KEY NOT NULL,
+	"b_name_farm" text,
+	"b_sector" "fdm-dev"."b_sector",
+	"created" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated" timestamp with time zone
+);
+--> statement-breakpoint
+CREATE TABLE "fdm-dev"."fertilizer_aquiring" (
+	"b_id_farm" text NOT NULL,
```

Comment:
spelling error? acquiring

Reply:
Resolved in 874a7f2830e1c2a776c7dd2c512f91321639df19